packages/demo/examples/08-network/renderer/DirectedLink.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const proptypes = {
+  linkStyles: PropTypes.object,
+  link: PropTypes.object.isRequired,
```

Comment:
should almost never use `object` because it's not specific, can you update to shape? (could define a shape in `@data-ui/network` and import it)